src/nyc_trees/js/src/selectableBlockfaceLayer.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+
+var L = require('leaflet'),
+    zoom = require('./mapUtil').zoom,
+
+    color = '#FFEB3B';
+
+module.exports = {
+    create: createBlockfaceLayer
+};
+
+/* Adds a geojson and utfgrid layer to the given map, based on the geojson data
+ * of the utfGrid.
+ * GeoJSON fatures are added to the map when their corresponding
+ * features are clicked in the utfGrid, and they are removed when they geoJSON
+ * is clicked.
+ *
+ * Accepts two callbacks: onAdd and onRemove, which are called when prior to
+ * adding or removing geoJSON.
+ *
+ * If the callbacks do not return a truthy value, the feature will not be
+ * added/removed.
+ *
+ * Returns the created geojson layer
+ */
+function createBlockfaceLayer(options) {
+    var map = options.map,
+        gridUrl = options.gridUrl,
```

Comment:
Note to self: It is probably better to take a `UtfGrid` object, so other features on the grid can be interacted with (like showing popups for unavailable blockfaces)